gr-utils/python/modtool/core/add.py

```diff
@@ -143,38 +143,41 @@ def run(self):
 
     def _run_cc_qa(self):
         " Add C++ QA files for 3.7 API if intructed from _run_lib"
-        fname_qa_h  = 'qa_{}.h'.format(self.info['blockname'])
-        fname_qa_cc = 'qa_{}.cc'.format(self.info['blockname'])
+        blockname_=self.info['blockname']
+        fname_qa_h  = f'qa_{blockname_}.h'
+        fname_qa_cc = f'qa_{blockname_}.cc'
```

Comment:
it's however, very appropriate to use a specific variable `blockname_` here, because that's just way more readable below, where it's used a couple of times :)